pages/[id].js

```diff
@@ -47,11 +55,25 @@ export default function DetailsPage({ sortedDefaultTasks }) {
       <StyledDetailsPage>
         <StyledTaskFlexWrapper>
           <StyledPriority $priority={currentTask.priority} />
-          <StyledDate $dateColor={pastDueDate ? "red" : "black"} />
+          <StyledDate $dateColor={pastDueDate ? "red" : "black"}>
+            {currentTask.dueDate}
+          </StyledDate>
         </StyledTaskFlexWrapper>
         <h3>{currentTask.title}</h3>
         <StyledDescription>
           <p>{currentTask.description}</p>
+          <button type="button" onClick={() => setIsOpen(true)}>
+            Delete
+          </button>
+          {isOpen && (
+            <Modal
+              onClose={closeModal}
+              onConfirm={(event) => handleConfirm(event, dynamicId)}
+            >
+              <p>Are you sure you want to delete this task?</p>
+            </Modal>
+          )}
+
           <StyledBackLink href="/">&larr;</StyledBackLink>
```

Comment:
I´d place this "back" Button a bit on the left above corner, instead of "poniting" to the left on the "Delete"-Button. No show stopper.

Reply:
Thanks for your idea, when we work on the task we will definitely think about your ideas